### DISCUSSION TITLE
Two important things that would be heavily desireable..

### DISCUSSION THREAD

**Discussion by lonewulf47** [flybywiresim/aircraft#5907] [Ideas]
Gentlemen,
I would recommend two things to be introduced in future builds:

1. Selectable database. Many Users are working with Jeppesen data (navigraph), which are much more accurate and worlwide compared to the provided NavBLUE
2. Saving of custom states. Powering up from cold and dark is sometimes really a PITA, if you do it more than once per day. Remember: IRL hand-overs from one crew to another are usually done in a fully powered turnaround state, just requiring route-specific inputs

**Comment by tracernz** [comment]
> Selectable database. Many Users are working with Jeppesen data (navigraph), which are much more accurate and worlwide compared to the provided NavBLUE

Navigraph provide a tool to replace the Navblue data with their own in MSFS. In any case, stay tuned on this one...

**Reply by lonewulf47** [reply]
I'm sorry, I'm not aware of that. I didn't see antyhing similar on their Homepage. I only know the Navdatabase that needs to be entered into the \Community folder. Unfortunately this does not provide the ultimate solution, as the conceptual error of missing (or wrong) priority of the Navdatabase towardsAdd-On airports ist still maintained. This is an error that is based on missing knowledge about AIRNC data...
